sdk/keyvault/azure-keyvault-certificates/samples/hello_world.py

```diff
@@ -57,19 +59,24 @@
 )
 cert_name = "HelloWorldCertificate"
 
+# [START create_a_certificate]
 # begin_create_certificate returns a poller. Calling result() on the poller will return the certificate
 # as a KeyVaultCertificate if creation is successful, and the CertificateOperation if not. The wait()
 # call on the poller will wait until the long running operation is complete.
 certificate = client.begin_create_certificate(
     certificate_name=cert_name, policy=cert_policy
 ).result()
 print(f"Certificate with name '{certificate.name}' created")
+# [END create_a_certificate]
 
+# [START get_certificate]
 # Let's get the bank certificate using its name
 print("\n.. Get a certificate by name")
 bank_certificate = client.get_certificate(cert_name)
 print(f"Certificate with name '{bank_certificate.name}' was found'.")
```

Comment:
This sample is particularly weird since it's playing out a scenario of managing bank information, which would make the README snippets pretty confusing.

In general, though, including the logging information that's included in samples for interactivity probably wouldn't be ideal to include in README snippets either. (Maybe we should get rid of them altogether? Probably not, though.)

Reply:
I removed "bank" from the samples. :)